README.md

```diff
@@ -8,14 +8,11 @@
 - This Github repository explores the website of a Basic Calculator made by Yoshitha Rathnayake.
 - This website shows a preview of my Calculator. 
 - The link for downloading the Calculator app is also available on the website.
-- The programming languages used to build this website are- HTML, CSS and BOOTSTRAP.
+- The programming languages used to build this website are- HTML, CSS, JAVASCRIPT and BOOTSTRAP.
 
 <a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a> 
-<a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>
+<a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a> <a href="https://www.w3schools.com/js/"><img src= "https://img.icons8.com/color/48/javascript.png"/><a> 
```

Comment:
Remove this JS logo also @Zweak only name requested

Reply:
@YoshithaRathnayake done